ironfish/src/account/accounts.ts

```diff
@@ -431,6 +431,11 @@ export class Accounts {
     return decryptedNotes
   }
 
+  async transactionIsSynced(hash: TransactionHash): Promise<boolean> {
```

Comment:
[UNKNOWN] This will have to be implemented in a fast way as it replaces checking the `seenGossipFilter` so it will probably get called a lot